pkgs/development/python-modules/pycolmap/default.nix

```diff
@@ -0,0 +1,59 @@
+{
+  lib,
+  buildPythonPackage,
+  setuptools,
+  cmake,
+  colmap,
+  boost,
+  numpy,
+  scikit-build-core,
+  ruff,
+  ninja,
+  pybind11,
+  pybind11-stubgen,
+  clang-tools,
+  perl,
+}:
+buildPythonPackage {
+  pname = "pycolmap";
+
+  inherit (colmap) src version;
+
+  patches = colmap.patches ++ [ ./remove-version-bounds.patch ];
```

Comment:
Add comment how how this patch should be maintained, e.g. 

> To update this with newer versions, simply remove all version bounds for those lines that have them.

if that's correct.